src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const populationList = document.querySelectorAll('.population');
+const populationArrey = [];
+
+for (let i = 0; i < populationList.length; i++) {
+  let item = document.querySelectorAll('.population')[i].textContent;
+
+  populationArrey.push(item = +item.replaceAll(',', ''));
+}
+
+let totalPopulation = 0;
+
+totalPopulation = populationArrey.reduce((accumulator, currentValue) => {
+  return accumulator + currentValue;
+});
```

Comment:
1. I don't see you change totalPopulation somewhere else, so why not use const?
2. It`s better to set initial value for reduce
```suggestion
const totalPopulation = populationArrey.reduce((accumulator, currentValue) => {
  return accumulator + currentValue;
}, 0);
```